app/pubspec.yaml

```diff
@@ -57,7 +57,10 @@ dependencies:
       url: https://github.com/acterglobal/flutter_matrix_html.git
   flutter_slidable: ^3.0.0
   quds_popup_menu: ^0.0.1+1
-  acter_avatar: 2.0.0+5
+  acter_avatar:
+    git:
+      url: https://github.com/acterglobal/acter-avatar.git
+      ref: main
```

Comment:
hmm... shouldn't we release that as a new version?

Reply:
Yeah, I think I'd probably do that now. Couldn't manage time before few days...